Scripts for Fig5 maps

efluet opened the pull request:
This pull request adds a folder with data, scripts and maps for fig 5

brieanne-forbes commented:
@efluet  are the files in the data subfolder duplicate files that can be found elsewhere? If so, can you remove them and update your scripts to point to the original source? If not, can you explain how they differ?

brieanne-forbes commented:
@efluet Also can you please update the language from respiration to O2 consuption?

efluet commented:
> @efluet are the files in the data subfolder duplicate files that can be found elsewhere? If so, can you remove them and update your scripts to point to the original source? If not, can you explain how they differ?

The files I used were taken from the separate repo: [https://github.com/parallelworks/dynamic-learning-rivers](https://github.com/parallelworks/dynamic-learning-rivers). 
I am not sure if those are now duplicated in this repository.   Making this repository self contained seem ideal, but duplicating data from another one repo isn't.  Were there plans on combining the two repos?

efluet commented:
> @efluet Also can you please update the language from respiration to O2 consumption and write the units like the screenshot below? If formatting the sub/super scripts is difficult, I can fix it in illustrator. <img alt="image" width="223" height="64" src="https://private-user-images.githubusercontent.com/108300528/476076933-b6ccab6b-425a-4105-85e8-5c4ff674c72b.png?jwt=eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NTUwNTM3MDAsIm5iZiI6MTc1NTA1MzQwMCwicGF0aCI6Ii8xMDgzMDA1MjgvNDc2MDc2OTMzLWI2Y2NhYjZiLTQyNWEtNDEwNS04NWU4LTVjNGZmNjc0YzcyYi5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjUwODEzJTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI1MDgxM1QwMjUwMDBaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT1kZDdlZDAzOWRmM2UyNDI3YTk4NTg3ZGJjNDA1ZWRiNTE3ZDY5OGNmN2VkZTZjN2E5ZDE0MWQ5MzI3MDIwNDYxJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.uDQ3OXDdm3C0HnEaMNLWOmI9bEwM6O8UvWd75egIqVg">

In the latest commit, I updated the units to the figure legend and updated the maps themselves to include the latest ones with James' comments.  I had not updated the manually edited version with the new maps.  
The file you'll want to integrate with the other panels is: iconmodex_fig2_2panels_v05_manmod.pdf

brieanne-forbes commented:
Thanks for making these changes! @efluet 

We are not planning on combining the two repos but they will be published together. However, I am pretty sure [ICON-ModEx_Combined_Predicted_Observed_Respiration_Rates.csv](https://github.com/WHONDRS-Hub/ICON-ModEx_Open_Manuscript/blob/origin/fig5_maps/fig-sample-iteration-map/data/ICON-ModEx_Combined_Predicted_Observed_Respiration_Rates.csv) is from this repo because there are no files in dynamic-learning-rivers with that file name. Additionally, I am wondering if you can remove the two files that are from dynamic-learning-rivers and read them in directly with the url? You will have to go the file in the repo and right click on the "raw" button to copy that link. Example: 
`read_csv('https://github.com/WHONDRS-Hub/ICON-ModEx_Open_Manuscript/raw/refs/heads/origin/fig5_maps/fig-sample-iteration-map/data/ICON-ModEx_Combined_Predicted_Observed_Respiration_Rates.csv')`

For the units in the maps, it looks like one "-1" is missing after "sediment". 

Additionally, can you please add your contact information to your scripts. 

In summary, here are my proposed changes:
- [ ] Remove ICON-ModEx_Combined_Predicted_Observed_Respiration_Rates.csv from your data subfolder 
- [ ] Update the script using the file above to the path within the repo
- [ ] Remove ICON-ModEx_Data.csv and post_03_output_all_sites_avgpre_stdpre_merged.csv from your data subfolder 
- [ ] Update the script using the files above to directly read the files from the dynamic-learning-rivers repo
- [ ] Add "-1" after "sediment" in units within the figures 
- [ ] Add contact information to the scripts

efluet commented:
I deleted the file:  post_03_output_all_sites_avgpre_stdpre_merged.csv
It turns out it wasn't even used in the final version of scripts (maybe replaced by a newer file; can't remember)

Anyway- my numerous bite-sized commits should cover it all (sorry! I am bad at this XD )

brieanne-forbes commented:
Thank you @efluet! I sent a teams message but putting a checklist here for next requests 

- [ ] check that candidate sites are supposed to be the same size as sampled sites in iconmodex_fig2_2panels_v05_manmod2.pdf
- [ ] delete unused figs 
- [ ] check remaining figs and update units 
- [ ] update link to correct data file in the repo 
- [ ] DELETE DATA FOLDER
